README.md

```diff
@@ -10,11 +10,11 @@ This repository is configured to run [`arista.cvp`](https://github.com/aristanet
   <img src='docs/imgs/cv_ansible_logo.png' alt='Arista CloudVision and Ansible'/>
 </p>
 
-To access an ATD topology, please contact your Arista representative.
+To access an ATD topology, please get in touch with your Arista representative.
 
 ## Lab topology
 
-The diagram below shows that the ATD lab topology consists of two Spines, four Leafs, and two Hosts.
+The diagram below shows that the ATD lab topology has two data centers. We will only leverage DC1 in this example.
```

Comment:
The only counter here is this single DC repository always felt like a quick demo for customers. If you add the second DC this would lead to increased build and deploy time. Although, that makes sense to include more use cases in this repository like DCI. Ill add this as an issue